net/ftpproxy/src/opnsense/mvc/app/models/OPNsense/FtpProxy/FtpProxy.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+/**
+ *    Copyright (C) 2016 EURO-LOG AG
+ *
+ *    All rights reserved.
+ *
+ *    Redistribution and use in source and binary forms, with or without
+ *    modification, are permitted provided that the following conditions are met:
+ *
+ *    1. Redistributions of source code must retain the above copyright notice,
+ *       this list of conditions and the following disclaimer.
+ *
+ *    2. Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *
+ *    THIS SOFTWARE IS PROVIDED ``AS IS'' AND ANY EXPRESS OR IMPLIED WARRANTIES,
+ *    INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY
+ *    AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE
+ *    AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY,
+ *    OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *    SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *    INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *    CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *    ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *    POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+namespace OPNsense\FtpProxy;
+
+use OPNsense\Base\BaseModel;
+
+/**
+ * Class FtpProxy
+ * @package OPNsense\FtpProxy
+ */
+class FtpProxy extends BaseModel
+{
+	/**
+	 * map config to ftp-proxy flags
+	 * and set default values
+	 * @param $node configuration
+	 * @return string
+	 */
+	public function configToFlags($node)
```

Comment:
@AdSchellevis @fbrendel let's talk on Friday when we all have a bit of time. I also want the package install/deinstall scripts to disappear, but it requires a design choice about plugging in the anchors I don't want to do alone. When both issues are resolved this can go into the 16.7.6 release already maybe :)